### DISCUSSION TITLE
Support data links for cells in any gauge display mode

### DISCUSSION THREAD

**Discussion by KBac** [grafana/grafana#38229] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
Support data links for cells in any gauge display mode

**Why is this needed**:
I use gauge display mode for columns. And I need to add data links to related dashboards into those columns. But they don't appear. They appear only if I change the display mode to "color text" or "color background".

**Comment by KBac** [comment]
Any plans to support data links for gauge display modes? Still doesn't work in Grafana 8.0.6.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
